CHANGELOG.md

```diff
@@ -5,9 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog], and this project adheres to
 [Semantic Versioning].
 
+## [0.5.0] - 2023-05-11
```

Comment:
Wrong month.

Maybe make it clear people can use the old version if they need the old feature.

Reply:
Doh! I'll fix the month and make a note per your suggestion.